### PR TITLE
Add Stremio

### PR DESCRIPTION
**Pull request by Ivshti**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Some good apps written with Electron.
 - [Visual Studio Code](https://code.visualstudio.com) - Free cross-platform IDE from Microsoft.
 - [Nylas](https://www.nylas.com/blog/splitting-the-atom) - Email client.
 - [Prepros](https://prepros.io) - Compile almost any preprocessing language with live browser refresh.
+- [Stremio](http://www.strem.io) - The next generation media center.
```
Comment (sindresorhus) [inline readme.md]: > Next generation media center.

**Comment by sindresorhus** [comment]
Can you include evidence for it using Electron?

**Comment by Ivshti** [comment]
@sindresorhus thanks for the fast response. Under what form? 

Would this do it: 
screen shot 2015-08-23 at 19 45 22

**Comment by sindresorhus** [comment]
:+1:

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: @Ivshti

Comment (Ivshti) [inline readme.md]: Done

**Comment by Ivshti** [comment]
@sindresorhus changed description

**Pull request merged**